src/server.js

```diff
@@ -35,7 +35,8 @@ module.exports = function() {
 		.engine("html", expressHandlebars({
 			extname: ".html",
 			helpers: {
-				tojson: (c) => JSON.stringify(c)
+				tojson: (c) => JSON.stringify(c),
+				translate: (c) => i18n.t(c)
```

Comment:
Client JS needs to perform most of the translations. Only initial Loading/enable js strings need to be translated.